docs/Privacy/Privacy-Quickstart.md

```diff
@@ -36,7 +36,7 @@ To run this tutorial, you must have the following installed:
 
 ## Clone Pantheon Quickstart Source Code
 
-Clone the repository from the `pantheon-quickstart` repository where `<version>` is replaced with the latest version (for example, `1.1.0`). 
+Clone the repository from the `pantheon-quickstart` repository where `<version>` is replaced with the latest version (for example, `{{ versions.quickstart }}`). 
```

Comment:
Maybe we could directly use the same thing as the regular quickstart : 
```
    Clone the repository from the `pantheon-quickstart` with the version matching Pantheon version, currently `{{ versions.quickstart }}`:

    ```bash tab="Linux/MacOS"
    git clone --branch {{ versions.quickstart }} https://github.com/PegaSysEng/pantheon-quickstart.git
    ```
```

Reply:
Updated